service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/SendAckTemplateParams.java

```diff
@@ -0,0 +1,17 @@
+package uk.nhs.adaptors.gp2gp.ehr;
```

Comment:
Would you say this class belongs here:
```suggestion
package uk.nhs.adaptors.gp2gp.ehr.model;
```